docs/csaf_provider.md

```diff
@@ -106,6 +106,12 @@ The following example file documents all available configuration options:
 # If a list item starts with `expr:`
 #   the rest of the string is used as a JsonPath expression
 #   to extract a string from the incoming advisories.
+#   If the result of the expression is a string this string
+#   is used. If the result is an array each element of
+#   this array is tested if it is a string or an array.
+#   If this test fails the expression fails. If the
+#   test succeeds the rules are applied recursively to
+#   collect all strings in the result.
```

Comment:
Please add something like the following to the documentation
```suggestion
#   collect all strings in the result.
#   Suggested expressions are:
#   - vendor, product family and product names:  "expr:$.product_tree..branches[?(@.category=='vendor' || @.category=='product_family' || @.category=='product_name')].name"
#   - CVEs: "expr:$.vulnerabilities[*].cve"
#   - CWEs: "expr:$.vulnerabilities[*].cwe.id"

Reply:
Addressed by bf6dfafffdf79f2697e7a00912fe09e337b8b3ee

Reply:
Lines 115 to 120 are a duplicate of lines 109-114.

Reply:
> Lines 115 to 120 are now a duplicate of lines 109-114.

Addressed by 69dda45bac1ae7ae4733a3b4c0e9caa8d160cb51